pkgs/top-level/python-packages.nix

```diff
@@ -6732,7 +6752,9 @@ in {
     inherit (pkgs.darwin.apple_sdk.frameworks) Foundation Security;
   };
 
-  tensorflow = self.tensorflow-build;
+  tensorflow = self.tensorflow-1;
+  tensorflow-1 = self.tensorflow-build-1;
```

Comment:
We use underscores for version numbers in attribute names.
```
tensorflow = self.tensorflow_1;  # 2, I don't know
tensorflow_1 = ...;
tensorflow_2 = ...;
```

Reply:
I would go with `tensorflow = self.tensorflow_1`, as it doesn't break people who are still on 1

Reply:
I'd prefer to default to the more recent version. I think some slight inconvenience for people that are still stuck at 1 is acceptable, and I'd rather have the current version as the "blessed path".

If we decide to drop `tf1` at some point, that would also be easier if `tf2` was the default.

Reply:
@timokau I'd to merge this branch to both 20.03 and master, in which case it should be non-breaking, so I'll do the bump after.